packages/react-native-reanimated/apple/reanimated/apple/REAModule.mm

```diff
@@ -151,12 +151,10 @@ - (void)sendEventWithName:(NSString *)eventName body:(id)body
   WorkletsModule *workletsModule = [_moduleRegistry moduleForName:"WorkletsModule"];
   auto jsCallInvoker = _callInvoker.callInvoker;
   auto jsiRuntime = reinterpret_cast<facebook::jsi::Runtime *>(self.bridge.runtime);
-  auto isBridgeless = ![self.bridge isKindOfClass:[RCTCxxBridge class]];
 
   assert(jsiRuntime != nullptr);
 
-  auto reanimatedModuleProxy =
-      reanimated::createReanimatedModule(self, self.bridge, jsCallInvoker, workletsModule, isBridgeless);
+  auto reanimatedModuleProxy = reanimated::createReanimatedModule(self, _moduleRegistry, jsCallInvoker, workletsModule);
```

Comment:
Replaced `bridge` with `_moduleRegistry` as we used to do in `createReanimatedModuleBridgeless` before it was removed.